docs/recipes/WritingTests.md

```diff
@@ -79,7 +79,7 @@ export default function todos(state = initialState, action) {
   switch (action.type) {
   case ADD_TODO:
     return [{
-      id: state.length,
+      id: state.reduce((maxId, todo) => Math.max(todo.id, maxId), -1) + 1,
```

Comment:
Initially `maxId` comes from the seed argument to the `reduce(fn, seed)`. In this case the seed is `-1`. Then, inside the `reduce